backend/dataall/modules/datasets/indexers/location_indexer.py

```diff
@@ -1,4 +1,5 @@
 """Indexes DatasetStorageLocation in OpenSearch"""
+
```

Comment:
This is because of the linting right? It was also flagged for me while executing ruff locally

Reply:
Yes, it's linting.